ES/ESap009.xml

```diff
@@ -375,20 +517,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <particDesc>
-            <listPerson>
-               <person>
-                  <persName role="owner" ref="PRS0000">The donor was possibly Ba-Ḫāyla ʾIyasus
-                     mentioned in Additio 1.</persName>
-                  <!--ACHTUNG! This is actually paragraphic information identifying the donor. 
-                                        Person/persName should only contain the person name, not all the paragraph.-->
-               </person>
-               <person>
-                  <persName ref="PRS00000"/>
-                  <!--other-->
-               </person>
-            </listPerson>
-         </particDesc>
+         
          <textClass>
             <keywords scheme="#ethioauthlist">
                <term key="RitualsAndRites"/>
```

Comment:
all manuscripts belonging to the Christian tradition should have the keyword ChristianLiterature